<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TEToolMainWindow</class>
 <widget class="QMainWindow" name="TEToolMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title Examining Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="exportCSVAction"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="imporCSVAction"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="connectAction"/>
    <addaction name="disconnectAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionconnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actiondisconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="databaseConnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="databaseDisconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="connectAction">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="disconnectAction">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="exportCSVAction">
   <property name="text">
    <string>to CSV</string>
   </property>
  </action>
  <action name="imporCSVAction">
   <property name="text">
    <string>from CSV</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
